content/sensu-core/1.6/changelog.md

```diff
@@ -53,6 +54,38 @@ menu: "sensu-core-1.6"
 - [Core 0.23.1 Release Notes](#core-v0-23-1)
 - [Core 0.23.0 Release Notes](#core-v0-23-0)
 
+## Core 1.6.0 Release Notes {#core-v1-6-0}
+
+Source: [GitHub.com][63]
+
+**October 12, 2018** &mdash; Sensu Core version 1.6.0 has been released
+  and is available for immediate download. Please note the following
+  improvements:
+
+### CHANGES {#core-v1-6-0-changes}
+
+- **NEW**: The [Checks API][64] now provides an endpoint to delete check history and
+  check results, giving you an easy way to clean up unused checks.
+
+- **IMPROVEMENT**: Sensu Core 1.6 reduces the impact of RabbitMQ flow control by
```

Comment:
Here's my attempt:

> Sensu Core 1.6  implements changes in RabbitMQ communication by using two discrete connections to the transport instead of two channels on a single connection, thereby doubling the number of concurrent connections RabbitMQ will receive. This change prevents check result processing rates from being impacted by check execution request publishing rates and reduces the possibility of spurious keepalive alerts under certain scenarios.